src/lib/EnsResolver.ts

```diff
@@ -61,12 +66,9 @@ export default class EnsResolver {
     const record: any = {};
 
     const resolvedTablenames = await Promise.all(
-      tableNames.map((tableName) => {
-        return new Promise((resolve) => {
-          (async () => {
-            resolve([tableName, await this.resolveTable(tableName)]);
-          })();
-        });
+      tableNames.map(async (tableName) => {
+        tableName = tableName.trim().replace(/^\[/, "").replace(/\]$/, "");
+        return [tableName, await this.resolveTable(tableName)];
```

Comment:
We also have to remove the square brackets here since the map we are creating should not contain escape characters.  This is because we are going to use the map as the second argument to the sqlparser.normalize function, which does not expect the escape characters to exist in the map.
Example code:
```
const tableNames = await globalThis.sqlparser.getUniqueTableNames(
  "select * from [foo.bar.eth];"
); //  -> [ "[foo.bar.eth]" ]

const ensResolvedName = await this.resolveTable(tableNames[0]);
// -> "my_table_31337_123"

const nameMap = {
  "foo.bar.eth": ensResolvedName
};

const normalized = await globalThis.sqlparser.normalize(
  "select * from [foo.bar.eth];",
  nameMap
);
// -> { statements: ["select * from [my_table_31337_123]"], tables: ["my_table_31337_123"], type: "read" }
```